src/actions/user.js

```diff
@@ -68,19 +77,10 @@ export const authFailure = () => ({ type: AUTH_FAILURE });
 export const authorizeToken = () => (dispatch: Dispatch) => {
```

Comment:
Currently, the process of verifying user - _where_ and _when_ to  check that  -  is quite clumpsy implemented. It works, though :see_no_evil: Thoughts about this is appreciated

Reply:
First make it work, then make it pretty if there's itime)